intelisl_midas_v2.md

```diff
@@ -15,25 +15,18 @@ featured_image_2: no-image
 accelerator: cuda-optional
 ---
 
-```python
-import torch
-midas = torch.hub.load("intel-isl/MiDaS", "MiDaS")
-midas.eval()
-```
-
-will load the MiDaS v2.1 model. The model expects 3-channel RGB images of shape ```(3 x H x W)```. Images are expected to be normalized using
-`mean=[0.485, 0.456, 0.406]` and `std=[0.229, 0.224, 0.225]`. 
-`H` and `W` need to be divisible by `32`. For optimal results `H` and `W` should be close to `384` (the training resolution). 
-We provide a custom transformation that performs resizing while maintaining aspect ratio. 
-
 ### Model Description
 
-[MiDaS](https://arxiv.org/abs/1907.01341) computes relative inverse depth from a single image. The model has been trained on 10 distinct dataset using 
-multi-objective optimization to ensure high quality on a wide range of inputs.
-
+[MiDaS](https://arxiv.org/abs/1907.01341) computes relative inverse depth from a single image. The repository provides multiple models that cover different use cases ranging from a small, high-speed model to a very large model that provide the highest accuracy. The models have been trained on 10 distinct dataset using 
+multi-objective optimization to ensure high quality on a wide range of inputs. An overview of 
 
 ### Example Usage
 
+Install dependencies
+```python
+!pip install timm
```

Comment:
This won't work. Please move it to https://github.com/pytorch/hub/blob/master/scripts/install.sh :D